README.md

```diff
@@ -1,176 +1,59 @@
-  billing_provider_id: 14,
-  first_name: "Johnny",
-  last_name: "Smith",
-  address: address,
-  ssn: "123456789",
-  gender: "Male"
-)
-
-pt.save
-=> true
-
-# inspect errors
-pt = DaisybillApi::Models::Patient.new(billing_provider_id: 14, first_name: "Johnny")
-pt.save
-pt.errors.full_messages
-=> ["Last name can't be blank"]
-
-# Get list of injuries for a patient
-patient = DaisybillApi::Models::Patient.find(345)
-patient.injuries
-
-# Create Injury
-injury = DaisybillApi::Models::Injury.new(patient_id: 345, description: 'Broken Arm', claim_number: '1234567')
-injury.save
-
-# List of Bills for an Injury
-injury = DaisybillApi::Models::Injury.find(333)
-injury.bills
-
-# Get a specific Bill
-DaisybillApi::Models::Bill.find(345)
-
-# Create Bill
-bp = DaisybillApi::Models::BillingProvider.find(14)
-pos = bp.places_of_service.first
-rp = bp.rendering_providers.first
-
-sli = DaisybillApi::Models::ServiceLineItem.new(
-  procedure_code: "S9981",
-  units: 1,
-  modifier_codes: ["93", "17"],
-  diagnosis_code_1: "72700",
-  custom_unit_charge_cents: 12345
-)
-
-bill = DaisybillApi::Models::Bill.new(
-  injury_id: 345,
-  date_of_service: '2015-01-01',
-  rendering_provider_id: rp.id,
-  place_of_service_id: pos.id,
-  diagnosis_codes: ["72700"],
-  service_line_items: [sli]
-)
-
-bill.save
-
-# Create Bill Submission for Bill
-bill = DaisybillApi::Models::Bill.find(386315)
-bill_submission = DaisybillApi::Models::BillSubmission.new(bill_id: bill.id)
-bill_submission.create
-
-# Get list of bill payments for bill
-bill = DaisybillApi::Bill.find(1)
-bp = bill.bill_payments.first
-
-# Create attachment for Bill
-DaisybillApi::Models::Attachment.create(
-  bill_id: 5600,
-  report_type: "09",
-  document: File.open("path/to/file")
-)
-
-# Destroy Bill
-bill.destroy
+and run
+```bash
+$ bundle install
 ```
 
-# Pagination
-
-`Model.all` returns a `PageableCollection` object which includes enumberable.
-
-```ruby
-@page = injury.bills
+## Usage
+For usage examples, checkout the [DaisyBill API documentation site](http://dev.daisybill.com)
 
-@page.first_page
-@page.last_page
-@page.next_page
-@page.previous_page
+You can also view the [API reference](http://www.rubydoc.info/github/daisybill/daisybill_api/master)
 
-until @page.next_page.nil? do
-  puts @page.map(&:id)
-  @page = @page.next_page
-end
-```
+## Compatibility
+This gem is tested against the following versions of Ruby:
+- 1.9.3
+- 2.0.0
+- 2.1.5
+- 2.2.0
 
 ## Contributing
-1. Fork the project.
-2. Create a topic branch and preface with issue number if available. E.g, `4_undefined_method_name_for_nil_class`
-3. Implement feature or bug fix
-4. If specs were not added in step 3, add then now.
-5. Run `bundle exec rspec spec`.
-6. Commit and push your changes.
-7. Submit a pull request. Please do not include changes to the gemspec or version file.
+1. Fork the project
+2. Create a topic branch and preface with issue number if available:
+  * e.g. `4_undefined_method_name_for_nil_class`
```

Comment:
@Surzhko disregard, thought you were mentioning the README on GitHub. Fixed by @benliscio